aiinfra-cluster/installation_scripts/run_tests.sh

```diff
@@ -0,0 +1,84 @@
+declare -a TEST_FILES=(
```

Comment:
If the tests are working currently, then may be we can add it to the PR cloudbuild yaml ? that way we get coverage in this PR as well.

Reply:
I added a test stage to the dockerfile and add two more steps -- a `docker build` and `docker run` -- to the cloudbuild. Result [here](https://pantheon.corp.google.com/cloud-build/builds;region=global/85088e01-87b7-4dfb-ba9a-f0ffc99528fd?e=13803378&jsmode=O&mods=monitoring_api_prod&project=gce-ai-infra)